packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -273,7 +273,7 @@ export const SubscriptionCreate = ({
 
         <div
           className={classNames(
-            'product-payment rounded-lg tablet:rounded-t-none',
+            'product-payment shadow-sm shadow-grey-300 rounded-lg tablet:rounded-t-none',
```

Comment:
I think it's okay if we even made this `component-card product-payment tablet:rounded-t-none`.